deploy.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+docker pull $dockerUsername/todo-app-prod:latest
+docker tag $dockerUsername/todo-app-prod:latest registry.heroku.com/ss-todo-app/web
+echo "$HEROKU_API_KEY" | docker login --username="$HEROKU_USERNAME" --password-stdin registry.heroku.com
+docker push registry.heroku.com/ss-todo-app/web
+heroku container:login
```

Comment:
These are the wrong way around. You need to log into the image registry (`heroku container:login`) before you are allowed to push to it (`docker push registry.heroku.com/...`)

But fortunately you are manually logging in with `docker login` on the line before.

Still, I'd simplify this to just
```suggestion
heroku container:login
docker push registry.heroku.com/ss-todo-app/web
```